siriuspy/siriuspy/fofb/util.py

```diff
@@ -39,15 +40,15 @@ def set_default_psconfig_mat(option=None):
 
     _corr_currloop_kp = _corrs_dev.currloop_kp
     _corr_currloop_ti = _corrs_dev.currloop_ti
-    _corr_accfilter_gain = _corrs_dev.fofbacc_filter_gain
 
     mat = _np.zeros(
-        (_const.nr_chcv, _const.psconfig_nr_coeffs_columns + 3),
+        (_const.nr_chcv, _const.psconfig_nr_coeffs_columns +
+         _const.PSCONFIG_COEFF_FIRST_COL),
         dtype=float
     )
     mat[:, _const.PSCONFIG_KP_COL] = _corr_currloop_kp
     mat[:, _const.PSCONFIG_TI_COL] = _corr_currloop_ti
-    mat[:, _const.PSCONFIG_FILTER_GAIN_COL] = _corr_accfilter_gain
+    mat[:, _const.PSCONFIG_FILTER_GAIN_COL] = unit_gain
```

Comment:
Why do we want to save the matrix with unit gain instead of using the configured one?

Reply:
As we talked offline, this function currently covers the case of using the predefined filters. If we want to get filters from what's currently configured, we can add a new option, and that one would use the configured filter gains instead of unit gain.